src/python/WMCore/WMSpec/StdSpecs/StdBase.py

```diff
@@ -1079,6 +1079,9 @@ def getWorkloadCreateArgs():
                      "MergedLFNBase": {"default": "/store/data"},
                      "UnmergedLFNBase": {"default": "/store/unmerged"},
                      "DeleteFromSource": {"default": False, "type": strToBool},
+
+                     # Rucio rule subscription lifetime (used in ContainerRules by T0)
+                     "DatasetLifetime": {"default": 0, "type": int, "optional": True, "validate": lambda x: x >= 0},
```

Comment:
These lines should be removed from this creation spec `getWorkloadCreateArgs`. They are already defined in the `getWorkloadAssignArgs` method, meaning those are meant to be parameters that we only define when a workflow gets assigned (which isn't very meaningful for T0 purposes by the way, but it's completely different for central production).

Reply:
Ahh finally I understood when you said only an assignment parameter. Thanks for the clarification Alan.